cypress/support/pages/PlaceAnOrder.pageObject.js

```diff
@@ -0,0 +1,123 @@
+import PageObject from '../PageObject';
+
+class PlaceOnOrderPageObject extends PageObject {
+  url = '/index.html';
+
+  get emailField() {
+    return cy.get('#email');
+  }
+
+  get nameField() {
+    return cy.get('#name');
+  }
+
+  get countryField() {
+    return cy.get('#country');
+  }
+
+  get cityField() {
+    return cy.get('#city');
+  }
+
+  get messageAllert() {
+    return cy.get('.sweet-alert');
+  }
+
+  get creditCardField() {
+    return cy.get('#card');
+  }
+
+  get sendMessageBtn() {
+    return cy.contains('.btn', 'Send message');
+  }
+
+  get BtnForAddedProduct() {
+    return cy.get('[onclick="addToCart(9)"]');
+  }
+
+  get BtnForConfirmOrder() {
+    return cy.contains('[type=button]', 'Place Order');
+  }
+
+  get confirmAllertBtn() {
```

Comment:
There is a typo in the method name 'confirmAllertBtn'. It should be 'confirmAlertBtn'.